src/core-manager/bitfield-rle.js

```diff
@@ -31,13 +31,10 @@ class State {
 encode.bytes = 0
 
 /**
- * @param {Uint32Array} bitfield
- * @param {Buffer} [buffer]
- * @param {number} [offset]
+ * @param {import('type-fest').TypedArray} bitfield
```

Comment:
Are you sure this is safe to do? I don't fully remember all of this code, but when writing it was specifically written for uint32Arrays - I think there are issues with endianess when accepting other options, given the modifications I made to the code.